misc/python/materialize/mzcompose.py

```diff
@@ -197,7 +196,7 @@ def __init__(self, repo: mzbuild.Repository, name: str):
             compose = yaml.safe_load(f)
 
         # Stash away sub workflows so that we can load them with the correct environment variables
-        self.yaml_workflows = compose.pop("mzworkflows", None)
+        self.yaml_workflows = compose.pop("mzworkflows", None) or {}
```

Comment:
This allows for a `mzcompose.yml` file wilth no `workflows` section.

Reply:
This can be more concisely expressed as:

```suggestion
        self.yaml_workflows = compose.pop("mzworkflows", {}})
```

Reply:
Thanks, done.